calico/Makefile

```diff
@@ -107,12 +113,6 @@ kubeval: _site
 	! grep -C3 -P "invalid|\t\*" filtered.out
 	rm filtered.out
 
-helm-tests: bin/helm values.yaml
```

Comment:
I actually just removed all of these tests. They were meant to test that the generated yaml files were valid, but now that we're not generating them on the fly they're not as useful any more, as well as being old and not very extensive.